opcua_tools/value_parser.py

```diff
@@ -99,61 +99,61 @@ def parse_singular_value(val, tagtype):
         else:
             stripped = None
         if tagtype == "SByte":
-            if stripped is not None and stripped != "":

```

Comment:
Have you checked these changes all the way up to model-builder tests using the packages locally?

Reply:
I've just double-checked it - works well in all the places (tools/mapper/MB).